metals/src/main/scala/scala/meta/internal/metals/FindTextInDependencyJarsRequest.scala

```diff
@@ -0,0 +1,6 @@
+package scala.meta.internal.metals
```

Comment:
```suggestion
package scala.meta.internal.metals.findfiles
```
maybe keep this and the provider in a separate package similar to how treeview is kept?

Reply:
done!